### PR TITLE
add UI for delete account

### PR DESCRIPTION
**Pull request by kevodwyer**

### CODE REVIEW

File: src/components/account/account.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            <span @click="close" class="close">&times;</span>
+
+          <div class="modal-header">
+            <h3>Delete Account</h3>
+          </div>
+
+          <div class="modal-body">
+              <h4>If you choose to proceed you will lose access to your account and data!</h4>
+              <h4>This action is not reversible</h4>
+              <h4>You must enter your password to confirm you wish to DELETE your Account</h4>
```
Comment (ianopolous) [inline src/components/account/account.html:14]: DELETE your account and all your data.

File: src/components/account/account.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <warning
+                v-if="showWarning"
+                v-on:hide-warning="showWarning = false"
+                :warning_message='warning_message'
+                :warning_body="warning_body"
+                :consumer_func="warning_consumer_func">
+        </warning>
+      <div class="modal-footer">
+          <slot name="footer">
+          <button class="btn btn-success" @click="deleteAccountAction">
```
Comment (ianopolous) [inline src/components/account/account.html:32]: can we make this button red?

File: src/components/account/account.html (hunk 1 above)
Comment (kevodwyer) [reply src/components/account/account.html]: done

File: src/components/account/account.html (hunk 2 above)
Comment (kevodwyer) [reply src/components/account/account.html]: done

**Pull request merged**